models/credmark/tokens/token.py

```diff
@@ -70,7 +114,7 @@ def run(self, input: Token) -> Price:
                 input=Token,
                 output=dict)
 class TokenHolders(Model):
-    def run(self, input: Token) -> dict:
+    def run(self, _input: Token) -> dict:
```

Comment:
is this underscored because it isn't implemented?

Reply:
yes. to keep pylint quiet for now.